src/modules/comment/CommentCount.js

```diff
@@ -0,0 +1,13 @@
+const CommentCount = () => {
+  const totalComments = document.getElementById('items');
```

Comment:
Great job team 🤝

- [ ] As the previous reviewers suggested, kindly make sure the comments counter updates instantly once a new comment is added. Right now the counter is not changing.